src/main/java/org/fundacionjala/fc/config/PropertiesReader.java

```diff
@@ -0,0 +1,90 @@
+package org.fundacionjala.fc.config;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * PropertiesReader class.
+ */
+public final class PropertiesReader {
+
+    private FileReader reader;
+    private Properties property;
+    private static final String PATH = "gradle.properties";
+    private static PropertiesReader singleInstance;
+
+    private PropertiesReader() {
+        try {
+            reader = new FileReader(PATH);
+            property = new Properties();
+            property.load(reader);
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * get instance or create a new one.
+     * @return PropertiesReader instance.
+     */
+    public static PropertiesReader getInstance() {
+        if (singleInstance == null) {
+            singleInstance = new PropertiesReader();
+        }
+        return singleInstance;
+    }
+
+    /**
+     * get the BaseUrl from the file.properties.
+     * @return base url.
+     */
+    public String getBaseUrl() {
+        return property.getProperty("baseUrl");
+    }
+
+    /**
+     * get the User from the file.properties.
+     * @return User value.
+     */
+    public String getUser() {
+        return property.getProperty("user");
```

Comment:
Please consider changing `user` by `username` since you updated it in `gradle.properties` file